Breeze.UI/electron-builder.json

```diff
@@ -11,7 +11,7 @@
       "target":[
          "nsis"
       ],
-      "artifactName":"${productName}.v${version}.setup.${os}.${env.arch}.${ext}"
+      "artifactName":"${productName}_v${version}_${os}${env.arch}_setup.${ext}"
```

Comment:
In Windows, it's not very common to have underscores in name of files.
I think the old name is good for now.

Reply:
:+1: